packages/@core.next/apollo.jsx

```diff
@@ -52,7 +52,7 @@ let createApolloClient = (initialState, ctx) => {
 
 // On the client, we store the Apollo Client in the following variable.
 // This prevents the client from reinitializing between page transitions.
-let globalApolloClient = null
+export let globalApolloClient = null
```

Comment:
It's also is not a part of a public API. Why do you want to open it? It's better to use `useApolloClient`